src/lib/chat/matrix-client.ts

```diff
@@ -102,8 +102,10 @@ export class MatrixClient implements IChatClient {
       .then((response) => response?.body || []);
   }
 
-  async searchMentionableUsersForChannel(_channelId: string, _search: string) {
-    return [];
+  async searchMentionableUsersForChannel(channelId: string, search: string) {
```

Comment:
Just to summarize what we discussed in standup today, I think we can probably just search in the "other members" field of Conversations. That should always be populated. (Note: Channels probably still needs to be api based for now due to the Sendbird implementation but we don't need to implement the Matrix version for Channels yet anyway)

Reply:
yep, I have added the `otherMembers` field in the matrix-client and using that to search users for mentions in both channel & conversation

![image](https://github.com/zer0-os/zOS/assets/33264364/549f0ced-39ca-4ef1-a6a3-fdeaef7da961)
![image](https://github.com/zer0-os/zOS/assets/33264364/c619a8f9-9905-44a5-b0eb-219ed06b4bd7)

Reply:
I am using the matrixId to display the name of the user for now, and the sendbird implementation is untouched. 

Follow ups would be
- map the `otherMembers` in the channelInfo to the zOS users.
- update the sendbird implementation for "conversations" to use otherMembers as cache as well.

Reply:
Don't bother updating the sendbird implementation.